src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -328,7 +328,7 @@ public async Task<bool> DeleteSecretAsync(string secretName, string keyScope, Sc
                 return secrets;
             }, secretsFactory).ContinueWith(t =>
             {
-                if (t.IsFaulted && t.Exception.InnerException is KeyVaultErrorException)
+                if (t.IsFaulted && t.Exception.InnerException is RequestFailedException)
```

Comment:
I think this is the equivalent in the new library but I'm not 100% sure, will need to confirm through testing as it is not well documented

Reply:
Seems to be equivalent based on the validation testing

Reply:
discussed offline, leaving as is as there is clear logging on what the issue is